src/main/app/src/app/zaken/zaken-verdelen-dialog/zaken-verdelen-dialog.component.ts

```diff
@@ -38,23 +42,51 @@ export class ZakenVerdelenDialogComponent implements OnInit {
     }
 
     ngOnInit(): void {
+        this.groepFormItem = this.mfbService.getFormItem(new AutocompleteFormFieldBuilder().id('groep')
+                                                                                           .label('groep')
+                                                                                           .optionLabel('naam')
+                                                                                           .options(this.identityService.listGroepen())
+                                                                                           .build());
+        //this.filterFormItem = this.mfbService.getFormItem(new CheckboxFormFieldBuilder().id('filter')
```

Comment:
todo toevoegen of commentaar verwijderen

Reply:
done